mmdet3d/models/middle_encoders/sparse_encoder.py

```diff
@@ -404,7 +416,8 @@ def calculate_pts_offsets(self, points, boxes):
                         points[j][2] - (boxes[i][2] + boxes[i][2] / 2.0))
         return pts_indices.cpu(), center_offsets.cpu()
 
-    def aux_loss(self, points, point_cls, point_reg, gt_bboxes):
+    def aux_loss(self, points: Tensor, point_cls: Tensor, point_reg: Tensor,
```

Comment:
Whether get aux loss from a middle_enoder module normal? I think middle_encoder is a 3d backbone. So we need to put get aux loss logic to a head module?

Reply:
The input of aux loss is the temp variables in `middle_encoder` and they are not passed into the following pipeline like other head loss: `backbone` and `neck`. Also, as a characteristic of SASSD, it deeply binds with the middle_encoder to learn better point encoding.  Maybe it's unnecessary to add an aux_head directly upon the middle_encoder.